### DISCUSSION TITLE
Bitwise Operators in Programming | by Ankoji Durgesh | May 16, 2024 | SERVER X 101

### DISCUSSION THREAD

**Discussion by giscus** [SERVER-X-101/SERVER-X-101#22] [Articles]
# Bitwise Operators in Programming | by Ankoji Durgesh | May 16, 2024 | SERVER X 101

Bit stands for binary digit. A bit is the basic unit of information and can only have one of two possible values ":" 0 or 1.

https://www.serverx.org.in/articles/bitwise-operators-in-programming/

**Comment by bhargavmalasala** [comment]
Nice👏
